### DISCUSSION TITLE
Why is there no command to check the production file locally?

### DISCUSSION THREAD

**Discussion by TomokiMiyauci** [storybookjs/storybook#15579] [Help]
Many application frameworks provide the `serve` and `preview` commands.
I could not find them in the storybook documentation.

When using storybooks, I often set up a static file server for previewing the production code.

This is technically very easy to do, but it is useful.
Is there any reason why this feature is not provided as standard?

**Comment by marko-hologram** [comment]
I mean, should production be so much different than your development instance of Storybook? I guess not, so I personally don't see a reason for them to provide this command out of the box.

You could always just add a simple npm script that can do build production version and serve it with this with one command if you wish so:

```
"serve-storybook": "npm run build-storybook -- -o ./path/to/build && npx http-server ./path/to/build",
```

These commands are copied from https://storybook.js.org/docs/react/workflows/publish-storybook#build-storybook-as-a-static-web-application.

**Comment by TomokiMiyauci** [comment]
I didn't know that there was a single line of code that worked.

With this, there may be no need to provide a command.
